src/Countries/Slovenia.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Slovenia extends Country
+{
+    public function countryCode(): string
+    {
+        return 'si';
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'Novo leto' => '01-01', // New Year's Day
+            'Novo leto 2' => '02-01', // New Year's Day, yes it's a second day
```

Comment:
This should be `01-02` if the day is January 2nd, right?

Reply:
Thank you for noticing. In out country the default is  DD-MM that's why I accidentally put it this way.